build.gradle.kts

```diff
@@ -37,6 +37,8 @@ dependencies {
     runtimeOnly("io.jsonwebtoken:jjwt-jackson:0.12.6")
     testImplementation("org.mockito.kotlin:mockito-kotlin:5.2.1")
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation("com.stripe:stripe-java:29.4.0")
```

Comment:
Wow, I didn't expect you'd find this Stripe library 😮 👍  
Well done!

However, I'm not sure if we're using it.
If we're not, I think it would be better for us to remove it.